testutil/integration/evmos/network/params.go

```diff
@@ -4,24 +4,54 @@
 package network
 
 import (
+	"errors"
+
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
 	evmtypes "github.com/evmos/evmos/v16/x/evm/types"
 	infltypes "github.com/evmos/evmos/v16/x/inflation/v1/types"
 	revtypes "github.com/evmos/evmos/v16/x/revenue/v1/types"
 )
 
 func (n *IntegrationNetwork) UpdateEvmParams(params evmtypes.Params) error {
-	return n.app.EvmKeeper.SetParams(n.ctx, params)
+	// FIXME this will not work with sdk v0.50
+	// To make this work, will need to do it as in the real chain
+	// (submit gov proposal & vote for it)
```

Comment:
LoL, this new spec seems weird.
How about making helper methods that submit gov and vote for it.
Voting period is not matter because block time is 1 hour so we just need to loop commit until done.